Makefile

```diff
@@ -18,26 +43,16 @@ bucket:
 	           minio/mc \
 	           /bin/sh -c "chmod +x ./docker/minio/create-bucket.sh && ./docker/minio/create-bucket.sh"
 
-test:
-	docker-compose run composer ./vendor/bin/simple-phpunit
+##
+## Tests
+## -----
+##
 
-logs:
-	docker-compose logs -ft
+test: ## Run unit tests
```

Comment:
This is not consistent with the next entry, if the next one is `behat`, this one should be `phpunit`, and test might be some sort of alias or something.

Reply:
On this matter, I made the PR #125, I merge it to develop as soon as Travis is done, then you would be able to retrieve it and remade the commands accordingly

Reply:
yes but for me you doing wrong

test: phpunit behat 

phpunit:
	docker-compose run composer ./vendor/bin/simple-phpunit		

behat:
	docker-compose run php vendor/bin/behat

for me make more sense that way

Reply:
That work for me, the PR #125  is not merged yet, if you apply this change you can refuse mine that we will close without merging